rasa/core/policies/policy.py

```diff
@@ -353,6 +358,104 @@ def format_tracker_states(self, states: List[Dict]) -> Text:
         return "\n".join(formatted_states)
 
 
+class PolicyPrediction:
+    """Stores information about the prediction of a `Policy`."""
+
+    def __init__(
+        self,
+        probabilities: List[float],
+        policy_name: Optional[Text],
+        policy_priority: int = 1,
```

Comment:
I think a default of `1` is way better than a default of `None` as it's more robust. Developers tend to forget that it can `None` and then everything explods.

Reply:
wait, no. I think, it shouldn't have default value. `Policy._prediciton` method should set it to `Policy.priority`